torch/_functorch/_aot_autograd/autograd_cache.py

```diff
@@ -277,6 +291,27 @@ def __init__(
         self.disable_amp = torch._C._is_any_autocast_enabled()
         self.deterministic_algorithms = torch.are_deterministic_algorithms_enabled()
         self.autograd_config = config.save_config()
+        self.saved_tensors_hooks_fx_wrap_cache_hashes: tuple[list[str], list[str]] = (
```

Comment:
any reason to make these lists, given that we only support a single pack/unpack hook in the compiled graph region today?

Reply:
I used lists to be able to use them as output argument. Yes, we can change structure to `Optional(tuple[str, str])`